src/data.ts

```diff
@@ -20,7 +26,10 @@ export function collectDataFromConstructor (vm: Vue, Component: VueClass) {
   // create plain data object
   const plainData = {}
   Object.keys(data).forEach(key => {
-    if (data[key] !== undefined) {
+    if (
+      data[key] !== undefined
+      && (!meta || !meta.decoratorAwaredKeys[key])
```

Comment:
What if a user wants a field to be both in `data` and processed by decorator logic ?